diff -aurN openssh-6.3p1.orig/sshd_config openssh-6.3p1/sshd_config
--- openssh-6.3p1.orig/sshd_config	2013-09-13 22:39:36.174897171 +0200
+++ openssh-6.3p1/sshd_config	2013-09-13 22:47:41.202532231 +0200
@@ -72,7 +72,7 @@
 #PermitEmptyPasswords no
 
 # Change to no to disable s/key passwords
-#ChallengeResponseAuthentication yes
+ChallengeResponseAuthentication no
 
 # Kerberos options
 #KerberosAuthentication no
@@ -93,7 +93,7 @@
 # If you just want the PAM account and session checks to run without
 # PAM authentication, then enable this but set PasswordAuthentication
 # and ChallengeResponseAuthentication to 'no'.
-#UsePAM no
+UsePAM yes
 
 #AllowAgentForwarding yes
 #AllowTcpForwarding yes
@@ -101,7 +101,7 @@
 #X11Forwarding no
 #X11DisplayOffset 10
 #X11UseLocalhost yes
-#PrintMotd yes
+PrintMotd no
 #PrintLastLog yes
 #TCPKeepAlive yes
 #UseLogin no
